examples/mnist-pytorch/README.rst

```diff
@@ -23,108 +28,96 @@ Clone this repository, locate into this directory:
+   pip install fedn 
+   export FEDN_AUTH_SCHEME=Bearer
+   export FEDN_PACKAGE_EXTRACT_DIR=package
+   fedn run client -in client.yaml --name client1 --secure=True --force-ssl
 
-.. code-block::
 
-   bin/build.sh
+Upload the initial model to FEDn Studio project. The seed.npz file is created when you run start the client and is found in ./data/models/seed.npz
 
-You should now have a file 'package.tgz' and 'seed.npz' in the project folder. 
+The default traning and test data is found in ./data/clients/1/mnist.pt and can be changed to other partitions by exporting the environment variable FEDN_DATA_PATH.
+The default split into 2 partitions can be changed in client/data.py.
 
-Next we prepare the local dataset. For this we download MNIST data and make data partitions: 
+Quick start with docker-compose
+-------------------------------
 
-Download the data:
+Clone this repository, locate into this directory:
 
 .. code-block::
 
-   bin/get_data
-
+   git clone https://github.com/scaleoutsystems/fedn.git
+   cd fedn/examples/mnist-pytorch
 
-Split the data in 10 partitions:
+Start a pseudo-distributed FEDn network using docker-compose:
 
 .. code-block::
 
-   bin/split_data --n_splits=10
+   docker compose \
+    -f ../../docker-compose.yaml \
+    -f docker-compose.override.yaml \
+    up -d
+
+This starts up the needed backend services MongoDB and Minio, the API Server and one Combiner. As well as two clients. 
+You can verify the deployment using these urls: 
 
-Data partitions will be generated in the folder 'data/clients'.  
+- API Server: http://localhost:8092/get_controller_status
+- Minio: http://localhost:9000
+- Mongo Express: http://localhost:8081
 
-FEDn relies on a configuration file for the client to connect to the server. Create a file called 'client.yaml' with the follwing content:
+To check the client output logs, run:
 
 .. code-block::
 
-   network_id: fedn-network
-   discover_host: api-server
-   discover_port: 8092
+   docker logs fedn-client-1
 
-Make sure to move the file ``client.yaml`` to the root of the examples/mnist-pytorch folder.
-To connect a client that uses the data partition ``data/clients/1/mnist.pt`` and the config file ``client.yaml`` to the network, run the following docker command:
+It should be waiting for the configuration of the package.
+
+Create the package (compress client folder):
 
 .. code-block::
 
-   docker run \
-  -v $PWD/client.yaml:/app/client.yaml \
-  -v $PWD/data/clients/1:/var/data \
-  -e ENTRYPOINT_OPTS=--data_path=/var/data/mnist.pt \
-  --network=fedn_default \
-  ghcr.io/scaleoutsystems/fedn/fedn:master-mnist-pytorch run client -in client.yaml --name client1
+   bin/build.sh
+
+You should now have a file 'package.tgz'. 
 
-Observe the API Server logs and combiner logs, you should see the client connecting and entering into a state asking for a compute package. 
+You are now ready to use the API to initialize the system with the package.
```

Comment:
Add instructions for how to do that inline here. Or add link to the right place in docs.